packages/api/test/pin.spec.js

```diff
@@ -0,0 +1,211 @@
+/* eslint-env mocha, browser */
+import assert from 'assert'
+import { endpoint } from './scripts/constants.js'
+import * as JWT from '../src/utils/jwt.js'
+import { SALT } from './scripts/worker-globals.js'
+import { JWT_ISSUER } from '../src/constants.js'
+
+function getTestJWT (sub = 'test', name = 'test') {
+  return JWT.sign({ sub, iss: JWT_ISSUER, iat: 1633957389872, name }, SALT)
+}
+
+describe.only('POST /pins', () => {
+  it('should receive pin data containing cid', async () => {
+    const name = 'car'
+    // Create token
+    const token = await getTestJWT()
```

Comment:
Could this be moved to a `before` block and saved in the a variable in describe scope instead to avoid repetition?